lib/models/period_wrapper.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'period_wrapper.freezed.dart';
+part 'period_wrapper.g.dart';
+
+@freezed
+
+/// Contains all the details associated with a Period
+class Period with _$Period {
+  const factory Period(
+    /// The number of period units
+    @JsonKey(name: 'periodUnit') String unit,
+
+    /// The increment of time that a subscription period is specified in
+    @JsonKey(name: 'periodNumberOfUnits') int value,
```

Comment:
We also have a field `iso8601` in purchases-android. Should we add it here now that we are adding this file